src/plugins/home/server/services/sample_data/lib/sample_dataset_schema.ts

```diff
@@ -67,6 +67,7 @@ export const sampleDataSchema = schema.object({
   description: schema.string(),
   previewImagePath: schema.string(),
   darkPreviewImagePath: schema.maybe(schema.string()),
+  icon: schema.maybe(schema.string()),
```

Comment:
NIT: even if now only used internally, the term `icon` is usually used for `euiIconName`, which can be misleading. I would at least add a comment explaining what we're expecting here. From the PR, I see a relative path is accepted, can this also accepts an absolute url?

Reply:
discussed offline, will rename to `iconPath`